python-package/setup.py

```diff
@@ -28,6 +28,7 @@
       install_requires=[
           'numpy',
           'scipy',
+          'scikit-learn',
```

Comment:
That is right. Let me remove it. After this build is finished since I want to know what is wrong with the build, I will submit another patch for removing these two lines.